.github/workflows/link-check.yml

```diff
@@ -0,0 +1,22 @@
+name: Links
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Run Lychee Link Checker
+      uses: lycheeverse/lychee-action@v1
+      with:
+        args: --base . --verbose --no-progress './apps/nextra/**/*.mdx' './apps/nextra/**/*.html'
```

Comment:
We probably need to figure out how to ignore `localhost` links.

This is why the broken link checker wasn't blocking before

Reply:
Can you give an example of what you mean by a local host link (I know what it is but when would it arise in the docs?)

Reply:
Just take a look at the output from the PR run

Reply:
Oh lmfao i see thanks. @kshitjee maybe something like this https://github.com/lycheeverse/lychee/issues/1384#issuecomment-1970108569

Reply:
I think it ignores localhost links now, check latest workflow run